README.md

```diff
@@ -0,0 +1,145 @@
+# Kodein-sample-testing [![Build Status](https://api.travis-ci.org/Karumi/kodein-sample-testing.svg?branch=master)]
+(https://travis-ci.org/Karumi/kodein-sample-testing)
+
+The main goal of this of this repository is written a small example to show how use Kodein to provide different implementations for production code and testing code.
+
+##Description
+
+The idea is to use Kodein to have the possibility to replace part of our production code for mocking implementations, with the objective to have control about the test environment.
+
+For writing those tests we are going to use a property that Kodein give us, the ability to override dependencies in a graph.
+
+Our first problem is Android does give us the possibility to provide dependencies to our activities using the constructor. Because Activities constructors are not public and we need to use Kodein to replace those dependencies.
```

Comment:
I'd rewrite this as:

> Our first problem is Android does give us the possibility to provide dependencies to our activities using the constructor. Because Activities constructors are not public and we need to use Kodein to replace those dependencies.

as:

> As Android does not give us the possibility to provide dependencies by constructor, we need to use Kodein to replace those dependencies. This is the main issue we fix using "override dependencies" Kodein feature.